integration_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -Eemo pipefail
+# This script is meant to be run inside https://hub.docker.com/_/docker
+# See .github/workflows/ci.yml:18 for how to run this yourself
+
+# Wait for the docker daemon to start up
+for i in {1..30}; do
+    if docker ps; then
+        break
+    else
+        sleep 1
+    fi
+done
+if [ "$i" = '30' ]; then
+    echo "Docker did not start within 30 seconds, aborting"
+    exit 1
+fi
+
+# Start docuum in the background
+/artifacts/docuum-x86_64-unknown-linux-musl --threshold=13MB &
+sleep 1
+# Wait for docuum to start sleeping by checking the process state
+function wait {
+    # http://man7.org/linux/man-pages/man5/proc.5.html
+    DOCUUM_PID=$(pgrep docuum-x86_64-unknown-linux-musl)
+    for i in {1..60}; do
+        if [ "$(awk '{ print $3 }' /proc/${DOCUUM_PID}/stat)" = 'S' ]; then
```

Comment:
Maybe checking that `ps -p "$DOCUUM_PID"` succeeds is a little more straightforward?

Reply:
That is a neat trick! Unfortunately look like BusyBox `ps` is a little too limited:

```
bash-5.0# ps -p "$DOCUUM_PID"
ps: unrecognized option: p
BusyBox v1.31.1 () multi-call binary.

Usage: ps [-o COL1,COL2=HEADER]

Show list of processes

	-o COL1,COL2=HEADER	Select columns for display
```

Reply:
Oh wait even better: I should just do `$!` to refer to the first background process like I do in the last line of this file 🤦